gitmanager.py

```diff
@@ -5,7 +5,7 @@
 import time
 import json
 from datetime import datetime
-from threading import Lock
+import threading
```

Comment:
Why?  It's *technically* still OK to do the import that we did and just use `Lock` directly like we were, there's no reason to import the entire module unless some issue has come up thus far with it that requires this change and refactoring elsewhere.

Reply:
`threading` is used mainly to maintain uniform style with other file.